### ISSUE TITLE
Bring back Window switching mode from old homerow (with single letters)

### ISSUE THREAD

**Issue by bdiebold** [dexterleng/homerow#69]
The "old" homerow hat a great additional shortcut that allowed to switch between windows:

<img width="495" alt="image" src="https://github.com/dexterleng/homerow/assets/5969866/28dbaa2a-0219-4b1f-9668-91acbedc9b67">

This is a great productivity boost and it would be awesome to have this back in the new homerow.
Would be happy to pay / donate additionally for this - let me know where 🤟.

**Comment by mak1A4** [comment]
+1

**Comment by emilbryggare** [comment]
Yes, this was one of the best features. I currently use Scoot for that https://github.com/mjrusso/scoot but not even close to as convenient.
